### ISSUE TITLE
I can only grab the left side of the image

### ISSUE THREAD

**Issue by saintsouth** [A-l-y-l-e/Alyle-UI#116]
# :beetle: Bug report

### Describe the bug
I can only grab the left side of the image.

### Minimal Reproduction
https://alyle.io/components/image-cropper

### Exception or Error
none.

### Screenshots
![Screenshot from 2020-05-29 10-31-50](https://user-images.githubusercontent.com/15023642/83211327-cece9480-a197-11ea-9d5a-a6d8843a9eb7.png)

### Which versions of Angular, Alyle UI, OS, TypeScript, browsers are affected?

**Alyle UI Version:** 
`v3.2.7`

**Angular Version:**
<pre><code>

Angular CLI: 9.1.1
Node: 10.15.0
OS: linux x64

Angular: 9.1.1
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.901.1
@angular-devkit/build-angular      0.901.7
@angular-devkit/build-optimizer    0.901.7
@angular-devkit/build-webpack      0.901.7
@angular-devkit/core               9.1.1
@angular-devkit/schematics         9.1.1
@angular/cdk                       9.2.1
@angular/flex-layout               9.0.0-beta.29
@angular/material                  9.2.1
@angular/material-moment-adapter   9.2.1
@ngtools/webpack                   9.1.7
@schematics/angular                9.1.1
@schematics/update                 0.901.1
rxjs                               6.5.4
typescript                         3.8.3
webpack                            4.42.0
</code></pre>

**Comment by Enlcxx** [comment]
Hello, thanks for reporting. I am fixing.

**Comment by abhishek-g** [comment]
Hi , we are using version 1.9.9 and facing same issue. Can anyone help us out.

**Comment by angelfraga** [comment]
Wow, I was getting crazy, even I've thought the new Google Chrome version 83.x.x.x introduced some kind of bug since my colleague was able to run this without problem in a previous 83.x.x.x chrome version. (that is still quite funny  or the issue is flaky)

Same issue for version 2.9.7. I have fixed it by adding all as value for `pointer-events` in my global `styles.scss`. 

```
/* comment these lines in order to reproduce the issue. */
.y-img-container-ic > canvas {
  pointer-events: all !important;
}
```

https://stackblitz.com/edit/angular-alyle-image-cropper-issue

@Enlcxx , currently I am not able to upgrade to 3.x.x version since our app is not 100% prepared for angular 9 and typescript >= 3.6. Could we check if that workaround is enough in order to generate `v2.9.8`? Hopefully, for @abhishek-g we could apply the same fix in order to generate `1.9.12`

**Comment by Enlcxx** [comment]
@abhishek-g I can't believe Alyle UI version 1.x.x is still being used.

Version 3.2.8 has the fix.

@angelfraga, Something similar happened to me, on my computer the bug does not show, but on my Android phone it does. Your solution is partially valid, but you cannot use that selector, since it is dynamically generated and can change.

I can't generate new versions for the previous versions, so you can use this solution:

```css
/* Alternative solution for https://github.com/A-l-y-l-e/Alyle-UI/issues/116 */
ly-img-cropper canvas,
ly-cropping canvas {
  pointer-events: all !important
}
```

You can add it to your style sheet.

**Comment by angelfraga** [comment]
@Enlcxx thanks for the quick answer and the tip with the CSS workaround.

**Comment by abhishek-g** [comment]
@Enlcxx Issue with our app is , we are on angular 6 right now and don't have an easy migration path ahead of us. the css fix isnt working for me , so nned your help to which version can we safely upgrade to have this working again

**Comment by Enlcxx** [comment]
@abhishek-g I understand, but you have checked if the style is being applied, I just tried it in an old version, and it works.

I can make a fixed version (only of this bug) for Angular 6, since which version do you want me to do it? Previously said you are using 1.9.9

**Comment by abhishek-g** [comment]
Heyy I did check, that style was getting applied, but still it didn't work. Yes I am using 1.9.9 , it would be of great help if we can release a version for the same with this fix.

**Comment by abhishek-g** [comment]
Thanks in Advance.

**Comment by Enlcxx** [comment]
@abhishek-g I cannot create a version from 1.9.9, another version 1.9.10 already exists. I am going to create it from 1.9.11, so the version with the changes would be 1.9.12.

**Comment by Enlcxx** [comment]
@abhishek-g new version released 1.9.12.

**Comment by Enlcxx** [comment]
@angelfraga new version released 2.9.8
